src/scss/mega/articles.scss

```diff
@@ -1,18 +0,0 @@
-@if $o-header-is-silent == false {
```

Comment:
@i-like-robots I've moved this and the other small partial into the main mega `menu.scss` file as there seemed no specific reason to keep them separate and the `menu.scss` file wasn't too large for it to become unwieldy. Happy to revert the change though if there was a good reason behind it?